### DISCUSSION TITLE
Audio distorted on audio monitoring, missing on recording/stream

### DISCUSSION THREAD

**Discussion by FableFoxWeaver** [DistroAV/DistroAV#790] [Q&A - Support]
Hello,
I think I ran into a problem, but before opening an issue, maybe it's an easy fix and someone can help...

I just updated to OBS 28 and NDI 4.10.
The audio I'm getting through obs-ndi is distorted.

Lemme take a step back.
The audio at the source PC isn't distorted.
The audio arriving at the receiving PC isn't distorted on NDI Studio Monitor.
It's only distorted in OBS, so I'm confident it's got to be an issue with obs-ndi.

I tried multiple options on the OBS source (Hardware Acceleration, etc).

I also found that recording through OBS records no sound. Stream also complained about the lack of sound (not sound in general, just this particular OBS source). Unfortunately, I can't show the distortion, because I can't record it.
The sound is recognizable, but distorted. I noticed it as I opened a game and the scene was of a fire and at first I thought there was an electric wire in the game or something.

I tried updating NDI on the souce PC (from 5 to 5.5)

What should my next step be? (Besides never updating OBS again :D)

**Comment by imeiul** [comment]
I had the same problem.
I'm using this fork that didn't have the problem.
https://github.com/dev47apps/obs-ndi

**Reply by FableFoxWeaver** [reply]
Thank you. I found in the OBS forums a version 5.0.0 of obs-ndi. Where did it come from? I have no idea, but I tried it and the issue was gone. However, it proved to be much less stable and particularly over Wi-Fi, it kept stuttering and flashing. It was fine over wired connection, but it didn't make me trust it too much. Don't want to have to deal with such issues during a stream.

Eventually I reverted OBS to 27 (and all the plugins) and everything is working fine now, even over Wi-Fi.

Thank you for your reply. I'll have to keep it in mind if I lose my mind and decide to update to 28 again :D

**Comment by Wakapedia** [comment]
I had the same issue. Just noticed last night on stream after someone heard and mentioned it. Mine seemed to be right channel only?
https://www.twitch.tv/wakapedia/clip/CooperativeSquareWaterDoubleRainbow-4hUIcwCD1-vAPePR 
Tried out the same 4.9.2 obs ndi plugin fork from September 4th and the audio issue went away
https://github.com/dev47apps/obs-ndi/releases

**Reply by FableFoxWeaver** [reply]
Interesting. 
In my case, my viewers actually complained they couldn't hear the NDI source.
I was like "hey, does that sound like an electric crackling?" and people were like "bruh, it's silent". 
If I do try to update to OBS 28 again, I'll try that version. Though hopefully by then it's fixed and more stable.

OBS 28 sucks visually anyway :D 
I mean, it's prettier, but less functional as it forces me to scroll a lot more to see my scenes.

Thank you for your reply :)
